### ISSUE TITLE
Playground crashes if metadata is added at end of file

### ISSUE THREAD

**Issue by zrknlzr** [traindown/traindown-js#1]
**Summary**
I started recording a session with the [playground](https://traindown.com/playground/l). Tried adding metadata at the end of file and it brings down the whole website. I assume this is simply not accounted for atm but thought to report anyway.

**Reproduce**
Browser: Chromium 87
```
@ 1/1/2020

* This is a Traindown scratch pad
* Practice your knowledge of Traindown!

# Good time?: YES

Squat:
  500
  
# metadata
```
**Expected**
Notify about wrong syntax or similar

**Actual results**
Browser Tab freezes and doesn't respond anymore until the site crashes.

**Notes:**
Very cool project though. Looking foward to using it more.

**Comment by scottt2** [comment]
Note to future self or whomever:

The issue is that the lexer is getting stuck because it's just paying attention to the terminating `:`. There needs to be an audit of similar situations where we need to always be cognizant of `EOF` at any random spot in the input.

This will also be something to consider across the entire suite of Traindown libraries.

**Comment by scottt2** [comment]
Addressed in v1.0.1.

**Comment by scottt2** [comment]
Should now see this
![image](https://user-images.githubusercontent.com/1002266/107839643-77df0e80-6d6a-11eb-8e94-508a1dc619cb.png)
